test/perf/benchmarks/powershell-perf.csproj

```diff
@@ -0,0 +1,30 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="../../Test.Common.props" />
+
+  <PropertyGroup>
+    <Description>PowerShell Performance Tests</Description>
+    <AssemblyName>powershell-perf</AssemblyName>
+    <OutputType>Exe</OutputType>
+
+    <NoWarn>$(NoWarn);CS8002</NoWarn>
+    <SuppressNETCoreSdkPreviewMessage>true</SuppressNETCoreSdkPreviewMessage>
+
+    <PlatformTarget>AnyCPU</PlatformTarget>
+    <DebugType>portable</DebugType>
+    <DebugSymbols>true</DebugSymbols>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <DelaySign>true</DelaySign>
+    <AssemblyOriginatorKeyFile>../../../src/signing/visualstudiopublic.snk</AssemblyOriginatorKeyFile>
+    <SignAssembly>true</SignAssembly>
+  </PropertyGroup>
+
+  <ItemGroup>
```

Comment:
Can we add maybe a condition for changing between ProjectReference and PackageReference. The condition can be switched via a build parameter to dotnet build.

Reply:
I added a property `PERFLAB_TARGET_VERSION` to control that.
However, the awkward thing is that `dotnet run` doesn't seem to honor a setting like `/p:PERFLAB_TARGET_VERSION=7.2.0-preview.5`.
   Run the following code, and then inspect the `out.txt`, you will find that `p:PERFLAB_TARGET_VERSION` is never set.
   ```
   dotnet run "/property:PERFLAB_TARGET_VERSION=7.2.0-preview.5" -c release -v d --filter *scriptblock* --list flat > C:\arena\tmp\out.txt
   ```
   When running with `dotnet build`, the property is set:
   ```
   dotnet build "/property:PERFLAB_TARGET_VERSION=7.2.0-preview.5" -c release -v d > C:\arena\tmp\out2.txt
   ```

Reply:
@adityapatwardhan When you get a chance, can you please play with it and see if I'm missing something when using `/property` with dotnet run`?

Reply:
The project needs to target multiple target frameworks, so that we can easily run perf test against any version of PowerShell nuget packages after 7.0.
I currently ran into some problems with using `ProjectReference` along with multiple targets. Will update this PR once I resolve that.

Reply:
Perhaps it is better to implement as a script since we have to _clean_ up build every time we change test scenario.

Reply:
The issue I ran into is a bug in the .NET SDK, see https://github.com/dotnet/sdk/issues/17013
Now I make the project only targets `net6.0`, but is able to use different versions of the PS nuget packages.